srcIdl/perftest.idl

```diff
@@ -5,6 +5,7 @@
 
 const long MAX_SYNCHRONOUS_SIZE = 63000;
 const long MAX_BOUNDED_SEQ_SIZE = 63000;
+const long TMP_MICRO_MAX_UNBOUNDED_SEQ_SIZE = 1048576; // 1024*1024
```

Comment:
Did we find the solution for this at the end? Didn't you fixed the issue in the templates for rtiddsgen? @franciscoRTI

Reply:
I mean to pass the "unbounded" thing and so.

Reply:
Yes, I think we can remove that and leave the sequences without a size. Latest micro codegen correctly should support -stringSize and -sequenceSize, but I have not tested that.